packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -363,12 +363,18 @@ export class Span implements APISpan, ReadableSpan {
 
   private _isSpanEnded(): boolean {
     if (this._ended) {
-      diag.warn(
-        `Can not execute the operation on ended Span {traceId: ${this._spanContext.traceId}, spanId: ${this._spanContext.spanId}}`
-      );
+        const error = new Error(`Operation attempted on ended Span {traceId: ${this._spanContext.traceId}, spanId: ${this._spanContext.spanId}}`);
+        
+        diag.warn(
+            `Cannot execute the operation on ended Span {traceId: ${this._spanContext.traceId}, spanId: ${this._spanContext.spanId}}. Change log level to debug for stack trace.`,
+            error // Pass the error object as the second argument
+        );
+        
+        // Optionally, you can still log the stack trace for additional context
+        diag.debug(`Stack trace for ended span operation: ${error.stack}`);
```

Comment:
I think this one now does not provide extra value since the error is passed to the warn, we can safely remove it :)